src/3.2.1/whats-new/index.html

```diff
@@ -186,7 +186,7 @@
     </div>
     <div class="span4">
       <h4><i class="icon-terminal"></i> SCSS Support</h4>
-      A long term solution is now in place for SCSS support. Need SASS? Try <a href="http://sass-lang.com/docs/yardoc/file.SASS_REFERENCE.html#syntax">sass-convert</a>.
+      A long term solution is now in place for SCSS support. Need Sass? Try <a href="http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax">sass-convert</a>.
```

Comment:
Interesting enough, `http://sass-lang.com/docs/yardoc/file.Sass_REFERENCE.html#syntax` is giving a 404 :)

Please use  `SASS_REFERENCE` here, as it was before